### PR TITLE
docs: release process

### PR DESCRIPTION
**Pull request by skylenet**

### CODE REVIEW

File: docs/Release-Process.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Release process for Swarm
+
+This document describes the steps required to perform a new release.
+
+## Pre release
+
+1. Make sure that the most recent builds are green and that smoke tests are passing on the cluster.
```
Comment (nonsense) [inline docs/Release-Process.md:7]: I think the whole document is great. We should probably be a bit more explicit about what we mean with `smoke tests are passing on the cluster`. Perhaps we need a whole other document that describes what we consider a healthy commit ready to be released.

File: docs/Release-Process.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+2. Check if protocols should be bumped. (e.g [PR#1465](https://github.com/ethersphere/swarm/pull/1465))
```
Comment (nonsense) [inline docs/Release-Process.md:8]: This is also related to verifying that the commit we want to release is healthy.

Probably best to split the `Release process` document from a document describing how we test and what we consider stable release candidate.

**Review by nonsense — APPROVED** [review]

File: docs/Release-Process.md (hunk 1 above)
Comment (acud) [reply docs/Release-Process.md:7]: agree. we can setup a baseline criteria of:

passing with no failures for 24 hours with:
- [ ] 10mb file
- [ ] 20mb file
- [ ] 40mb file
(and add more as we progress)

**Review by acud — APPROVED** [review]
nice one @skylenet.
apart from maybe minor refinements to the definitions of the smoke tests passing this LGTM as a starting point :+1:

**Pull request merged**